version.txt

```diff
@@ -1 +1 @@
-0.5.0
```

Comment:
Let's not increase version for each PR. We will do it when we want to release.

Reply:
+1

Reply:
Reverted version bump